Exam/CT/src/exercice2/Launcher.java

```diff
@@ -0,0 +1,16 @@
+package exercice2;
+
+import java.awt.image.BufferedImage;
+
+import javax.swing.JFrame;
+
+
+public class Launcher {
+	public static void main(String[] args) throws Exception {
+		int[][] img = Util.getLennaGreenBuffer();
+		IRasterBuffer mask = new MaskedRaster(img);
+		BufferedImage  bImg = RasterFactory.getImage(mask);
```

Comment:
Why didn't you configure the mask to show only lower right quarter? :disappointed: